src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,31 @@
+package duke;
+
+import java.util.ArrayList;
+
+public class Parser {
+    public Parser(){}
+
+    /**
+     * Take in the String input and split into the 3 parts, namely
+     * the command, the title and extra_descriptions.
+     * @param input The input from the users.
+     * @return a String array that contains different components of the input.
+     */
+    public String[] splitIntoComponents(String input) {
+        ArrayList<String> list = new ArrayList<>();
```

Comment:
You may want to indicate that it is list for inputs?

Reply:
Thanks for the feedback. I have made the amendment you suggested. Thanks